lasagne/layers/pool.py

```diff
@@ -433,11 +430,8 @@ def get_output_for(self, input, **kwargs):
             if a > 1:
                 input_shape = input.shape
                 output_shape = self.get_output_shape_for(input_shape)
```

Comment:
Since `input_shape` is neither reused nor shorter than `input.shape`, I'd prefer
```python
output_shape = self.get_output_shape_for(input.shape)
```